src/AcmCountCard/AcmCountCard.test.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { axe } from 'jest-axe'
+import { AcmCountCard } from './AcmCountCard'
+
+describe('AcmCountCard', () => {
+    const shareAction = jest.fn()
+    const cardAction = jest.fn()
+
+    // Skeleton Card Tests
+
+    const SkeletonCard = () => {
+        return <AcmCountCard id="ACM Skeleton Card" loading={true} />
+    }
+
+    test('validates Skeleton Card renders', () => {
+        const { getByTestId } = render(<SkeletonCard />)
+        expect(getByTestId('ACM Skeleton Card')).toBeInTheDocument()
+    })
+
+    test('has zero accessibility defects', async () => {
+        const { container } = render(<SkeletonCard />)
+        expect(await axe(container)).toHaveNoViolations
+    })
+
+    // Suggested Search Card Tests
+
+    const SuggestedSearchCard = () => (
+        <AcmCountCard
+            id="ACM Suggested Search Card"
+            cardHeader={{
+                title: 'Workloads',
+                description: 'A pre-defined search to help you review your workloads',
+                actions: [{ text: 'Share' }],
+                onActionClick: () => {
+                    shareAction()
+                },
```

Comment:
This should be the same...
```suggestion
                onActionClick: { shareAction() },
```